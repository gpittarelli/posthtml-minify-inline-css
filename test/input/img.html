<img src="" alt="Something" style="vertical-align: middle;">
