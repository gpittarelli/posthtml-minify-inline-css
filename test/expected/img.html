<img src="" alt="Something" stlye="vertical-align: middle;">
